run_analysis.py

```diff
@@ -37,6 +41,120 @@ def update_generate(config, generate):  # {{{
     config.set('output', 'generate', generateString)  # }}}
 
 
+def run_parallel_tasks(config, analyses, configFiles, taskCount):
```

Comment:
That's fine.  @pwolfram could review if he wants to but it implements the design document that he approved...  It's hard to understand so I wouldn't necessarily recommend trying to review the code directly as long as the design seems okay.